benchmarks/src/main/java/org/questdb/PongMain.java

```diff
@@ -45,7 +45,11 @@ public static void main(String[] args) {
         // worker pool, which would handle jobs
         final WorkerPool workerPool = new WorkerPool(() -> 1);
         // event loop that accepts connections and publishes network events to event queue
-        final IODispatcher<PongConnectionContext> dispatcher = IODispatchers.create(dispatcherConf, new IOContextFactoryImpl<>(PongConnectionContext::new, 8));
+        final IODispatcher<PongConnectionContext> dispatcher = IODispatchers.create(
+                dispatcherConf,
+                new IOContextFactoryImpl<>(PongConnectionContext::new, 8),
+                PlainSocketFactory.INSTANCE
```

Comment:
nit: may be default this?

Reply:
This method is used in only a few places and it's not a part of the public API. Do you think we need to overload it considering this?

Reply:
not needed, agreed, was just a top of the head idea :)